debian/control

```diff
@@ -13,7 +13,7 @@ Build-Depends: debhelper-compat (= 13),
  pkgconf
 Maintainer: Kiwix team <kiwix@kiwix.org>
 Homepage: https://www.openzim.org/wiki/Libzim
-Standards-Version: 4.4.1
+Standards-Version: 4.6.2
 Rules-Requires-Root: no
 
 Package: libzim8
```

Comment:
```suggestion
Package: libzim9
```
This reference should be changed together.

Reply:
sorry, That a very dumm error my side. Thx.